File: AdvancePython/mysql_conn.py
import mysql.connector

mydb = mysql.connector.connect(
    host="localhost",
    user="root",
    passwd="",
    database="employee"
)

mycursor = mydb.cursor()

mycursor.execute("CREATE DATABASE IF NOT EXISTS employee")
mycursor.execute("CREATE TABLE IF NOT EXISTS info (id INT PRIMARY KEY AUTO_INCREMENT, name VARCHAR(255), addr VARCHAR(255))")
mycursor.execute('INSERT INTO info (name, addr) VALUES ("Virat", "India"), ("Rohit", "India"), ("Rachin", "New Zealand")')

mydb.commit()

print(mycursor.rowcount, "record inserted.")

mycursor.execute("SELECT * FROM info")
myresult = mycursor.fetchall()
print(myresult)

File: node.js/mysql_conn.js
let mysql = require('mysql');

let con = mysql.createConnection ({
    host: "localhost",
    user: "root",
    passwd: "",
    database: "mydb"
});

con.connect(function(err) {
    if(err) throw err;
    console.log("Connected!");
    con.query("CREATE TABLE IF NOT EXISTS customers (id INT AUTO_INCREMENT PRIMARY KEY, name VARCHAR(255), addr VARCHAR(255))", function(err, result) {
        if(err) throw err;
        console.log("Table created!");
    });
    con.query("INSERT INTO customers (name, addr) VALUES ('John Doe', 'USA')", function(err, result) {
        if(err) throw err;
        console.log("1 record inserted!")
    });
    con.query("SELECT * FROM customers", function(err, result, fields) {
        if(err) throw err;
        console.log(result);
    })
});